config/muxcable.py

```diff
@@ -257,3 +264,208 @@ def disable(port, target):
         sys.exit(CONFIG_FAIL)
     click.echo("loopback disable sucessful")
     sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hw mode on a port"""
```

Comment:
Suggest rewording to `Configure muxcable hardware directly" or similar.

Reply:
fixed